testing/test_config.py

```diff
@@ -46,6 +46,22 @@ def test_getcfg_empty_path(self):
         """correctly handle zero length arguments (a la pytest '')"""
         getcfg([""])
 
+    def test_setupcfg_uses_toolpytest_with_pytest(self, testdir):
+        p1 = testdir.makepyfile("def test(): pass")
+        testdir.tmpdir.join("setup.cfg").write(
+            textwrap.dedent(
+                """
+                [tool:pytest]
+                testpaths=%s
+                [pytest]
+                testpaths=ignored
+                """
+                % p1
+            )
+        )
+        result = testdir.runpytest(str(p1))
+        assert result.ret == 0
```

Comment:
Used `getcfg` in the test first, but it behaves different when not passing in `config`, so I figured it is better testing this end-to-end.